### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by blessedcoolant**
I've decided to cleanup the README (it's getting absurdly long) and start a documentation process that is easier to follow. This is going to be a long process but here are the initial set of changes.

You can check out the updated docs here: https://github.com/blessedcoolant/stable-diffusion/tree/docs-update

I'll keep pushing changes regularly over the next day or two and then squash them all into one update.

## Done

- [x] Moving all documentation to a new folder called **docs**.
- [x] Intro clean up
- [x] Create a full list of contributors to ensure everyone is credited properly
- [x] Initial Install Docs
- [x] Initial Feature Docs
- [x] Move all assets to Docs folder.
- [x] Format cleanup on all the existing docs

## Todo

- [ ] Update, fix language and formalize all existing documentation.

## Help
- [ ] Localization -- If you are willing to help translate the docs into different languages, you are more than welcome to lend a hand. Thank you.

### CODE REVIEW

**Comment by blessedcoolant** [comment]
@lstein If you have any thoughts, suggestions or ideas in regards to this, please let me know.

**Comment by blessedcoolant** [comment]
This is good to go for now. I will formalize all language across all the docs over time. @lstein

**Comment by tildebyte** [comment]
@blessedcoolant; Should we put [Stable-Diffusion-local-Windows.ipynb](https://github.com/lstein/stable-diffusion/blob/main/Stable-Diffusion-local-Windows.ipynb) (it's primary purpose is to help with installation, not serve as a runtime notebook) and the text from the [Easy-peasy-Windows-install](https://github.com/lstein/stable-diffusion/wiki/Easy-peasy-Windows-install) Wiki page into the docs folder as well?

**Comment by blessedcoolant** [comment]
> @blessedcoolant; Should we put [Stable-Diffusion-local-Windows.ipynb](https://github.com/lstein/stable-diffusion/blob/main/Stable-Diffusion-local-Windows.ipynb) (it's primary purpose is to help with installation, not serve as a runtime notebook) and the text from the [Easy-peasy-Windows-install](https://github.com/lstein/stable-diffusion/wiki/Easy-peasy-Windows-install) Wiki page into the docs folder as well?

Not sure about the book but the other page can do into the docs. Can this be integrated into the Windows Installation doc somehow? Or does this need to be a separate thing?

**Comment by tildebyte** [comment]
Then too... it's seeming more Wiki-like, but I'm not sure if we want to have the docs "tied up" in the Wiki or not - the Wiki is cool, but the 'docs' directory will travel with the repo (so if you're triple-firewalled like lstein, you can still have the docs with you).

I'm not sure what the right answer is here.

**Comment by blessedcoolant** [comment]
> Then too... it's seeming more Wiki-like, but I'm not sure if we want to have the docs "tied up" in the Wiki or not - the Wiki is cool, but the 'docs' directory will travel with the repo (so if you're triple-firewalled like lstein, you can still have the docs with you).
> 
> I'm not sure what the right answer is here.

I feel for now we should keep all the docs with the repo. They'd be accessible offline too.

**Comment by tildebyte** [comment]
> Can this be integrated into the Windows Installation doc somehow? Or does this need to be a separate thing?

I need to see what the Windows install page looks like in this PR so I can decide... the Easy-peasy guide is reeallly step-by-step detailed, so it's pretty long (and there's a separate page just for installing Git for Windows 🙄); we might end up with a Windows Install index page pointing out to other pages.

**Comment by blessedcoolant** [comment]
> I need to see what the Windows install page looks like in this PR so I can decide... the Easy-peasy guide is reeallly step-by-step detailed, so it's pretty long (and there's a separate page just for installing Git for Windows 🙄); we might end up with a Windows Install index page pointing out to other pages.

Won't be a problem. The more detailed the better. The idea of having extra detailed docs is that we can forward people raising issues that have been addressed to the docs.

Have a look at the repo and let me know if it can be integrated somehow. Either you can do it or I can do it. Both work.

File: README.md [hunk 1]
```diff
@@ -12,439 +12,91 @@
...
-text-to-image generator. This fork supports:
+# **Stable Diffusion Dream Script**
 
-1. An interactive command-line interface that accepts the same prompt
-   and switches as the Discord bot.
+This is a fork of [CompVis/stable-diffusion](https://github.com/CompVis/stable-diffusion), the open source text-to-image generator. It provides a streamlined process with various new features and options to aid the image generation process.
 
-2. A basic Web interface that allows you to run a local web server for
-   generating images in your browser.
+_Note: This fork is rapidly evolving. Please use the [Issues](https://github.com/lstein/stable-diffusion/issues) tab to report bugs and make feature requests._
```
Comment (tildebyte) [inline README.md:19]: "AND PLEASE ACTUALLY FILL OUT THE MOTHER-@(&#^$ TEMPLATE"

But nicer than that, of course 😂

Comment (blessedcoolant) [reply README.md:19]: Added this: "Be sure to use the provided templates. They will help aid diagnose issues faster."

**Comment by tildebyte** [comment]
Right, so the current Windows Install doc is structured pretty well.

If you want, you can just pull the Markdown out of the Wiki into two individual markdown docs, fix up the main link in https://github.com/blessedcoolant/stable-diffusion/blob/docs-update/docs/installation/INSTALL_WINDOWS.md, fix up the Git for Windows install link in the Easy-peasy page, and all will be well.

**Comment by blessedcoolant** [comment]
> Right, so the current Windows Install doc is structured pretty well.
> 
> If you want, you can just pull the Markdown out of the Wiki into two individual markdown docs, fix up the main link in https://github.com/blessedcoolant/stable-diffusion/blob/docs-update/docs/installation/INSTALL_WINDOWS.md, fix up the Git for Windows install link in the Easy-peasy page, and all will be well.

I suggest we let it be as it is for this particular push.

In a few days, I'll do a formalization pass on the docs so the language is consistent and professional across all the docs. While doing that, I'll try to reframe the Install_Windows doc by taking the best of both yours and the existing doc and make it good to go. There's no sense in having two docs for the same purpose in my opinion.

**Review by glonlas — APPROVED** [review]
LGTM. This documentation refactoring will be beneficial for the project. Any nits can be added later.

**Comment by lstein** [comment]
As usual, you folks are running ahead of me! I was planning to undertake a docs cleanup tomorrow morning. I'll pull this in this evening (have an all-day obligations today)

**Comment by blessedcoolant** [comment]
> As usual, you folks are running ahead of me! I was planning to undertake a docs cleanup tomorrow morning. I'll pull this in this evening (have an all-day obligations today)

All good. :) Split the workload. Take your time. No rush.

**Review by lstein — APPROVED** [review]
Fantastic work. I'll merge in and walk through the docs systematically.

**Pull request merged**